bot/exts/evergreen/wonder_twins.py

```diff
@@ -0,0 +1,49 @@
+import random
+from pathlib import Path
+
+import yaml
+from discord.ext.commands import Bot, Cog, Context, command
+
+with open(Path.cwd() / "bot" / "resources" / "evergreen" / "wonder_twins.yaml", "r", encoding="utf-8") as f:
+    info = yaml.load(f, Loader=yaml.FullLoader)
+    WATER_TYPES, OBJECTS, ADJECTIVES = info["water_types"], info["objects"], info["adjectives"]
```

Comment:
I think it should be better to do this in `__init__` function, as constants is meant to be static, but loading them from file makes them dynamic.